tests/Feature/FormMigration/Steps/TestRazorpayPerFormSettings.php

```diff
@@ -0,0 +1,54 @@
+<?php
+
+namespace Feature\FormMigration\Steps;
+
+use Give\FormMigration\DataTransferObjects\FormMigrationPayload;
+use Give\FormMigration\Steps\RazorpayPerFormSettings;
+use Give\Tests\TestCase;
+use Give\Tests\TestTraits\RefreshDatabase;
+use Give\Tests\Unit\DonationForms\TestTraits\LegacyDonationFormAdapter;
+
+/**
+ * @unreleased
+ */
+class TestRazorpayPerFormSettings extends TestCase
```

Comment:
We'll want to test that `canHandle` method to make sure this is only processed when necessary.

Reply:
Resolved in d66bd27e5f620f475f022f4c063225107fc86176